src/analyses/cfg_dominators.h

```diff
@@ -500,23 +565,40 @@ void cfg_dominators_templatet<P, T, post_dom>::initialise(P &program)
   cfg(program);
 }
 
-/// Pretty-print a single node in the dominator tree. Supply a specialisation if
-/// operator<< is not sufficient.
-/// \par parameters: `node` to print and stream `out` to pretty-print it to
+/// Pretty-print a single node. Supply a specialisation if operator<< is not
+/// sufficient.
+/// \param node: node to print
+/// \param out: output stream
 template <class T>
 void dominators_pretty_print_node(const T &node, std::ostream &out)
 {
   out << node;
 }
 
+/// Pretty-print a single node.
+/// \param node: node to print
+/// \param out: output stream
+template <>
 inline void dominators_pretty_print_node(
   const goto_programt::targett &target,
   std::ostream &out)
 {
   out << target->code.pretty();
 }
 
+/// Pretty-print a single node.
+/// \param node: node to print
+/// \param out: output stream
```

Comment:
Not sure if there's much meaning to parameter documentation if it doesn't carry more information than the type. Then again, it doesn't hurt either.